pages/views/examples/forms-select.njk

```diff
@@ -228,12 +228,12 @@ aside:
           </esl-select>
         </div>
 
+        <label for="color_field" class="form-label">Color</label>
         <esl-select class="form-group"
                     style="display: block"
                     pin-selected
                     placeholder="Color"
                     select-all-label="All Colors">
-          <label for="color_field" class="form-label">Color</label>
```

Comment:
Is that change a requirement for this update
(I just worried about compatibility because independently of beta status, it has prod usages)

Reply:
UPD: we can't do such type of change it should be changed by fixing the `activator` param inside `esl-select` to `this.$renderer,`

Reply:
Change was made because inconsistency in layout was noticed. For the first select-list example "Countries", label tag is on the same level as esl-select-list tag